cmd/influx/cli/cli.go

```diff
@@ -294,6 +294,9 @@ func (c *CommandLine) ParseCommand(cmd string) error {
 
 // Connect connects to a server.
 func (c *CommandLine) Connect(cmd string) error {
+	// normalize cmd
+	cmd = strings.ToLower(cmd)
```

Comment:
I was a little concerned about lowercasing the hostname in the rest of the string, but as far as I can research, host names are expected to be case insensitive in any context here.